src/platform-includes/getting-started-install/unreal.mdx

```diff
@@ -1,14 +1,22 @@
 The Unreal Engine (UE) SDK is officially supported for the three latest UE versions. However, it is likely to be compatible with older engine versions as well depending on the specific features and functionality that you need.
 
-To build the SDK, download the latest sources from the [Releases page](https://github.com/getsentry/sentry-unreal/releases) and place it in the project's `Plugins` directory. On the next project launch, UE will prompt to build Sentry module.
+To install the SDK, download the latest sources from the [Releases page](https://github.com/getsentry/sentry-unreal/releases) and place it in the project's `Plugins` directory. On the next project launch, UE will prompt to build Sentry module.
 
 <Note>
 
 Currently, this method is available only for C++ UE projects. Blueprint projects can be converted to a C++ one by adding an empty class using the editor.
 
 </Note>
 
-After the successful build, in the editor navigate to the **Settings > Plugins > Code Plugins** menu and check whether the Sentry plugin is enabled:
+<Note>
+
+[Releases page](https://github.com/getsentry/sentry-unreal/releases) provides two types of plugin packages (note `github` and `marketplace` suffixes in package names). The key difference between those two is the crash capturing backend which plugin uses under the hood on Windows. The UE Marketplace version relies on `Breakpad` (in-proc handler) that requires UE application or game to be relaunched in order to send the crash report to Sentry while the GitHub version uses `Crashpad` (out-of-proc handler) which does that right away.
+
+</Note>
+
+Alternatively, Sentry SDK can be downloaded from its UE Marketplace page via the Epic Games Launcher in which case the standard installation process will take place.
```

Comment:
Here we mention the alternative but it's not clear to me what I should be doing.